test/integration/benchmark/jsonify/main.go

```diff
@@ -72,7 +72,7 @@ func run() error {
 	data := PerfData{Version: "v1"}
 	for _, benchMarks := range benchmarkSet {
 		for _, benchMark := range benchMarks {
-			data.DataItems = appendIfMeasured(data.DataItems, benchMark, benchparse.NsPerOp, "time", "μs", benchMark.NsPerOp/1000.0)
+			data.DataItems = appendIfMeasured(data.DataItems, benchMark, benchparse.NsPerOp, "time", "us", benchMark.NsPerOp/1000.0)
```

Comment:
Thanks @oomichi 
Do we really need to replace it?  `μs` is easier to recognize as a unit of time.

Reply:
Ah, when I opened this file with `vim` that seems a Chinese character and I could not see the meaning.
I am fine to keep this char as we see it on the browser.

Reply:
@SataQiu Is current updated PR good for you?